src/game/option_passport.c

```diff
@@ -234,7 +381,9 @@ void Option_Passport(INVENTORY_ITEM *inv_item)
     case PASSPORT_PAGE_1:
         Text_Hide(m_Text[TEXT_LEFT_ARROW], true);
         if (m_PassportMode == PASSPORT_MODE_SHOW_SAVES) {
-            Option_PassportShowSaves();
+            Option_PassportLoadGame();
```

Comment:
`Option_PassportShowSaves` is now called within `Option_PassportLoadGame`. I did this to maintain the existing look and the new select level code added a bunch of new code and indentations.

Reply:
Good, you have good intuition of wanting to reduce the indentation levels.